apps/ledger-live-mobile/src/components/RootNavigator/types/EditTransactionNavigator.ts

```diff
@@ -0,0 +1,77 @@
+import { TransactionRaw } from "@ledgerhq/live-common/families/ethereum/types";
```

Comment:
❌  importing ethereum specific from a generic component

Reply:
How would solve this ? Knowing that I specifically want to import an ethereum `TransactionRaw`